### PR TITLE
Kitchen

### PR DESCRIPTION
**Pull request by UndeadNappist**
Adds butcher knives to the autolathe;
Nerfs butterfly knife damage by reducing their w_class and their force;
Makes Butcher Cleavers larger but reducing their force, making them harder to hide but probably hitting harder anyway.

### CODE REVIEW

File: code/game/objects/items/weapons/material/knives.dm [hunk 1]
```diff
@@ -7,8 +7,8 @@
 	var/active = 0
 	w_class = ITEM_SIZE_TINY
 	attack_verb = list("patted", "tapped")
-	force_divisor = 0.25 // 15 when wielded with hardness 60 (steel)
-	thrown_force_divisor = 0.25 // 5 when thrown with weight 20 (steel)
+	force_divisor = 0.10
```
Comment (Kurgis) [inline code/game/objects/items/weapons/material/knives.dm]: Can you list the damage value it has with steel?

File: code/game/objects/items/weapons/material/knives.dm [hunk 2]
```diff
@@ -56,7 +56,7 @@
 	flags = CONDUCT
 	sharp = 1
 	edge = 1
-	force_divisor = 0.15 // 9 when wielded with hardness 60 (steel)
+	force_divisor = 0.20 // 9 when wielded with hardness 60 (steel)
```
Comment (Kurgis) [inline code/game/objects/items/weapons/material/knives.dm]: Can you update the comment to reflect the changed value?

File: code/game/objects/items/weapons/material/knives.dm [hunk 3]
```diff
@@ -7,8 +7,8 @@
 	var/active = 0
 	w_class = ITEM_SIZE_TINY
 	attack_verb = list("patted", "tapped")
-	force_divisor = 0.25 // 15 when wielded with hardness 60 (steel)
-	thrown_force_divisor = 0.25 // 5 when thrown with weight 20 (steel)
+	force_divisor = 0.10
+	thrown_force_divisor = 0.15 // 5 when thrown with weight 20 (steel)
```
Comment (Kurgis) [inline code/game/objects/items/weapons/material/knives.dm]: Can you update the comment to reflect the changed value?

**Comment by Kurgis** [comment]
It should be noted that the butcher knife inherits the resource cost of a regular knife, and should have its own increased resource cost.

**Comment by UndeadNappist** [comment]
Fixed.

**Pull request merged**